src/controller/testController.ts

```diff
@@ -21,6 +22,8 @@ export function createTestController(): void {
     testController.resolveHandler = async (item: TestItem) => {
         await loadChildren(item);
     };
+
+    startWatchingWorkspace();
```

Comment:
what if I add folder into workspace?

Reply:
Good catch, I filed an issue to track this item. Would like to fix it systematically after all the components code are ready.